Dockerfile

```diff
@@ -6,18 +6,38 @@ RUN sudo apt-get update -y &&\
     sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 2 &&\
     sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 3 
 RUN python3 -m pip install --upgrade pip
+RUN echo "export ROS_PACKAGE_PATH=/src/aerialist/aerialist/inter_images:\$ROS_PACKAGE_PATH" >> /root/.bashrc
+
+RUN sudo apt install ros-melodic-stereo-image-proc ros-melodic-image-view -y
+# RUN sudo apt-get install python-rospy
 #&&\
 # sudo -H pip3 install --upgrade pip
 
 #Setting up the current tool 
 COPY ./requirements.txt /src/aerialist/requirements.txt
+WORKDIR /src/catkin_ws/src/avoidance/
+RUN catkin_create_pkg spawn_obstacles std_msgs rospy roscpp cv_bridge
+WORKDIR /src/catkin_ws/src/avoidance/spawn_obstacles/src
+RUN mkdir -p nodes
+WORKDIR /src/catkin_ws/src/avoidance/spawn_obstacles/src/nodes
+COPY aerialist/ros_node_script/spawn_script/ .
+RUN catkin build -w /src/catkin_ws &&\
+    echo "source /src/catkin_ws/devel/setup.bash" >> ~/.bashrc  
 WORKDIR /src/aerialist/
 RUN pip3 install -r requirements.txt
+RUN pip2 install roslibpy
+RUN pip3 install roslibpy
 COPY . .
 RUN chmod +x ./aerialist/__main__.py
 COPY ./template.env ./.env
 RUN mkdir -p /io/ ./results/logs/ tmp/
 
+RUN cd /home &&\
```

Comment:
Added the code to download all the required gazebo models from this repo. I need to add the gazebo model of tree and apartment to load it into the simulator.